### PR TITLE
Added hound config file

### LINKED ISSUE #5: Setup hound config file
- to ensure that PR matches eslint rules

### PR DESCRIPTION
**Pull request by MASTERCHIEF011**
Fixes #5

### CODE REVIEW

**Comment by MASTERCHIEF011** [comment]
is there any custom config for eslint that should be included?

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
please enable js

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+scss:
+  enabled: false
+
+go:
+  enabled: true
+
+javascript:
+  enabled: false
```
[the remark this answers is not in the document]
Comment (MASTERCHIEF011) [reply .hound.yml]: Sorry I overlooked that.

**Comment by YashKumarVerma** [comment]
add support for eslint / tslint checks

**Review by YashKumarVerma — APPROVED** [review]

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**